modes/xref/evil-collection-xref.el

```diff
@@ -50,13 +52,24 @@
 
     ;; open
     (kbd "RET") 'xref-goto-xref
+    (kbd "TAB") 'xref-quit-and-goto-xref
```

Comment:
This one might be unexpected by a user. Maybe another key?

Reply:
That's the default Emacs style binding:

``` elisp
(defvar xref--xref-buffer-mode-map
  (let ((map (make-sparse-keymap)))
    (define-key map (kbd "n") #'xref-next-line)
    (define-key map (kbd "p") #'xref-prev-line)
    (define-key map (kbd "N") #'xref-next-group)
    (define-key map (kbd "P") #'xref-prev-group)
    (define-key map (kbd "r") #'xref-query-replace-in-results)
    (define-key map (kbd "RET") #'xref-goto-xref)
    (define-key map (kbd "TAB")  #'xref-quit-and-goto-xref)
    (define-key map (kbd "C-o") #'xref-show-location-at-point)
    ;; suggested by Johan Claesson "to further reduce finger movement":
    (define-key map (kbd ".") #'xref-next-line)
    (define-key map (kbd ",") #'xref-prev-line)
    (define-key map (kbd "g") #'xref-revert-buffer)
    map))
```

Agree that it's confused, what about changing to `S-RET`?
`xref-quit-and-goto-xref` has already two bindings (`o` and `go`) to invoke.

Reply:
Maybe it's OK, I'm not sure if we have precedence yet for "action -> close current window".